### PR TITLE
Added test cases for addmeeting, deletemeeting, findtag, and others

### PR DESCRIPTION
**Pull request by isanidiot**

### CODE REVIEW

**Review by ReaganTan00 — APPROVED** [review]
Good test cases to increase coverage, lgtm

**Comment by shauneang** [comment]
wait ah got typo

File: src/test/java/seedu/address/logic/commands/AddMeetingCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.MeetingTime;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
+
+class AddMeetingCommandTest {
+
+    private static final String MEETINTTIME_STUB_1 = "10-10-2022-22:30";
```
Comment (shauneang) [inline src/test/java/seedu/address/logic/commands/AddMeetingCommandTest.java:32]: MEETINGTIME_STUB_1 and MEETINGTIME_STUB_2 spelt at MEETINTTIME

**Review by shauneang — CHANGES_REQUESTED** [review]

**Review by shauneang — APPROVED** [review]
lgtm

**Review by eugenetayyj — APPROVED** [review]
lgtm

**Pull request merged**